test/rops_indices.txt

```diff
@@ -997,4 +997,4 @@
 29433
 5088
 1713
-5038
\ No newline at end of file
+5038
```

Comment:
We can't see CRLF or CR line endings on Github. Can someone please verify that this is the correct diff? (I can't do it, sorry)

Reply:
I was actually going to propose to not touch data files like these.

Reply:
Time to test the tests 🤣

Reply:
I would say so. This is not our code. It's data.

Reply:
@mtochi please undo the changes in these two text files

Reply:
@kunaltyagi I did that, but it's still failing two checks, what could be the problem?